package.json

```diff
@@ -47,6 +47,7 @@
     "site": "bundle exec jekyll serve",
 
     "lint": "tslint -c tslint.json 'src/**/*.ts' 'test/**/*.ts'",
+    "lint:fix": "tslint -c tslint.json --fix 'src/**/*.ts' 'test/**/*.ts'",
```

Comment:
I think you can just run `npm run lint -- --fix` instead of adding a new command.